lib/plugins/aws/remove/lib/stack.js

```diff
@@ -26,8 +27,47 @@ module.exports = {
       .then(() => cfData);
   },
 
+  disassociateUsagePlan() {
```

Comment:
If this is an OK approach to remove the association from usage plan, I'll change first two requests (CloudFormation describeStackResource and APIGateway getUsagePlans) to be parallel so it should be a little bit faster.